src/plugins/FullScreenPlugin.js

```diff
@@ -0,0 +1,114 @@
+import { ButtonPlugin } from '../base-plugins/ButtonPlugin';
+import { Button } from '../ui-elements';
+
+/**
+ * A Springroll plugin to easily set up togglable fullscreen
+ */
+export class FullScreenPlugin extends ButtonPlugin {
+  
+  /**
+   *  Creates an instance of FullscreenPlugin
+   * 
+   * @param {string} targetElementSelector The selector for the element to be made fullscreen
```

Comment:
This conflicts with the documentation, which says this param can be an array of selectors.